.github/workflows/spki.yml

```diff
@@ -55,4 +55,7 @@ jobs:
           toolchain: ${{ matrix.rust }}
           override: true
       - run: cargo test --release
+      - run: cargo test --release --features fingerprint
+      # - run: cargo test --release --features alloc
```

Comment:
Why is this commented out?

Reply:
Because I forgot to remove it. Will fix. The `alloc` feature only covers the Base64 encoding at this point, there are no additional functionality or dependencies to be tested in `alloc` alone.

Reply:
Fixed in latest push